blocks/product-featured-image-block/features/product-featured-image/default.jsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+
+import { useFusionContext } from "fusion:context";
+import { Image } from "@wpmedia/arc-themes-components";
+import { RESIZER_APP_VERSION } from "fusion:environment";
+
+const BLOCK_CLASS_NAME = "b-product-featured-image";
+
+export const ProductFeaturedImageDisplay = ({ data, resizerAppVersion }) => {
+	const featuredImageAsset = data.schema?.featuredImage?.value?.assets.find(
+		(asset) => asset.type === "image"
+	);
+
+	if (!featuredImageAsset) {
+		return null;
+	}
+
+	const { url, auth, alt_text: altText } = featuredImageAsset;
```

Comment:
`alt_text` may not always be present on the data, are we accounting for that?

Reply:
the Image component is. But I can add a test to validate it if we want.

Reply:
updated